src/Func.h

```diff
@@ -833,7 +833,7 @@ class Func {
                               const std::vector<ExternFuncArgument> &params,
                               Type t,
                               int dimensionality) {
-        define_extern(function_name, params, Internal::vec<Type>(t), dimensionality);
+        define_extern(function_name, params, std::vector<Type>{t}, dimensionality);
```

Comment:
Resolved offline. For posterity: this is a workaround for the fact that it's ambiguous whether {t} is a copy-constructed Type or an initializer list containing a single Type. Not worried about the syntax because this case should never occur in user code.